aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/RegionDecorator.kt

```diff
@@ -211,6 +211,10 @@ class RegionProviderConfig(codegenContext: ClientCodegenContext) : ConfigCustomi
                     )
                 }
 
+                is ServiceConfig.BuilderFromConfigBag -> {
```

Comment:
Side-note: it would be nice to have the type system ensure we implement all these config injections for each field that gets added to config. Seems like it will be easy to forget one in the future.

Reply:
Yeah, that's important to ensure. Given the scope, added a tracking issue in 97cecd3 to be worked on separately.